.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "useTabs": false,
+  "printWidth": 120,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": false,
+  "bracketSpacing": true,
+  "endOfLine": "lf",
+  "arrowParens": "avoid"
+}
```

Comment:
This is cool, but now a lot of files in the repo are failing prettier's check. Create a separate ticket to fix the formatting in those files.